### DISCUSSION TITLE
Loading Screen (background)

### DISCUSSION THREAD

**Discussion by sercandemirtas** [google/model-viewer#1937] [Q&A]
Hello all,

Is it possible to make loading screen transparent ? As you can see in the picture and demo link, white screen appears with my loading gif before my model load. (ps loading gif is transparent). I want to remove white screen.

Thank you..

#### Live Demo

(Only desktop version is stable)

https://silly-ramanujan-97ec74.netlify.app/urun.html

<img width="1440" alt="Ekran Resmi 2020-11-27 00 08 58" src="https://user-images.githubusercontent.com/68244546/100391757-852b0d80-3045-11eb-9cc7-92b3fa19d93a.png">

### Browser Affected

- [x ] Chrome
- [ x] Edge
- [x ] Firefox
- [x ] IE
- [x ] Safari

**Comment by elalish** [comment]
Yes, just set the `--poster-color` CSS to a transparent color: https://modelviewer.dev/docs/#entrydocs-loading-css-posterColor
